topics/microbiome/tutorials/dada-16S/tutorial.md

```diff
@@ -0,0 +1,1050 @@
+   > 2. How many readsa are in F3D0 sample before Filter and Trim?
+   >
+   > > <solution-title></solution-title>
+   > >
+   > > In F3D0 element in the collection, we see:
+   > > 1. `reads.in`: 7793
+   > > 2. `reads.out`: 7113
+   > >
+   > {: .solution}
+   >
+   {: .question}
+
+To better see the impact of filtering and trimming, we can inspect the read quality profiles and compare to the profiles before.
+
+<div class="Step-by-step" markdown="1">
+
+> <hands-on-title>Inspect read quality profiles after Filter And Trim</hands-on-title>
+>
+> 1. {% tool [dada2: plotQualityProfile](toolshed.g2.bx.psu.edu/repos/iuc/dada2_plotqualityprofile/dada2_plotQualityProfile/1.28+galaxy0) %} with the following parameters:
+>    - *"Processing mode"*: `Joint`
+>        - *"Paired reads"*: `paired - in a data set pair`
+>            - {% icon param-collection %} *"Paired short read data"*: `Paired reads` output of **dada2: filterAndTrim**
+>        - *"Aggregate data"*: `No`
+>    - *"sample number"*: `10000000`
+>
+{: .hands_on}
+
+</div>
+
+> <question-title></question-title>
+>
+> Looking at outputs of **plotQualityProfile** for reverse reads before and after **dada2: filterAndTrim**
+> 
+> {% snippet faqs/galaxy/features_scratchbook.md %}
+>
+> ![Quality Profiles for every samples with read length on the X-axis and Quality score on the Y-axis](./images/plotQualityProfile_reverse.png "Quality profiles for reverse reads before trimming and filtering")
+>
+> ![Quality Profiles for every samples with read length on the X-axis and Quality score on the Y-axis](./images/plotQualityProfile_reverse_after_filter_trim.png "Quality profiles for reverse reads after trimming and filtering")
+>
+> How did trimming and filtering change the quality profiles for reverse reads?
+>
+> > <solution-title></solution-title>
+> >
+> > The reads have been trimmed making the profiles looking better. 
+> >
+> {: .solution}
+>
+{: .question}
+
+## Learn the Error Rates
+
+The DADA2 algorithm makes use of a parametric error model (`err`) and every amplicon dataset has a different set of error rates. The **dada2: learnErrors** method learns this error model from the data, by alternating estimation of the error rates and inference of sample composition until they converge on a jointly consistent solution. As in many machine-learning problems, the algorithm must begin with an initial guess, for which the maximum possible error rates in this data are used (the error rates if only the most abundant sequence is correct and all the rest are errors).
+
+<div class="Step-by-step" markdown="1">
+
+**dada2: learnErrors** tool expects as input one collection and not a paired collection. We then need to split to paired collection into 2 collections: one with forward reads and one with reverse reads.
+
+> <hands-on-title> Unzip paired collection </hands-on-title>
+>
+> 1. {% tool [Unzip collection](__UNZIP_COLLECTION__) %} with the following parameters:
```

Comment:
What I always do is to add name tags `#forward` and `#reverse` to the collection. Then it's much more difficult to mix up samples.

But renaming the datasets is fine as well.

Reply:
Good point. I modified the tutorial to use the tags